lib/charms/mongodb/v1/mongodb_tls.py

```diff
@@ -293,20 +314,51 @@ def _on_certificate_expiring(self, event: CertificateExpiringEvent) -> None:
 
         self.set_tls_secret(internal, Config.TLS.SECRET_CSR_LABEL, new_csr.decode("utf-8"))
 
-    def _get_sans(self) -> List[str]:
+    def get_new_sans(self) -> Dict:
```

Comment:
```suggestion
    def get_new_sans(self) -> dict[str, list[str]]:
```

we can probably get away with making this method protected `_get_sans`

Reply:
unfortunately this lib is owned by the VM mongodb charm. Is this a blocker?